docs/synctl-patch-test.md

```diff
@@ -45,7 +46,6 @@ synctl patch test id [options]
 
 ### Options for API script test
 ```
-    -f, --from-file <file-name>        load synthetic test payload from (.json) file
     --script <file>                    load script (.js) from file
     --bundle <bundle>                  set bundle content
```

Comment:
we can make it more clear like "Synthetic bundle test script, support zip file (.zip) path  or  zip file content encoded with base64